### ISSUE TITLE
Make zoom a per-page setting

### ISSUE THREAD

**Issue by tfuxu** [ranfdev/Geopard#109]
Instead of globally storing zoom level, it should be stored by each webview page (and it should be disabled completely in custom pages, eg. input page)

**Comment by ranfdev** [comment]
I'm ok with disabling the zoom on the custom pages, like the input one, but it doesn't make much sense to have a separate zoom for each gemini page. 

HTML browsers have different zoom levels for each page because each website has a custom font size. Here the font size is the same everywhere. Why would a user zoom page X but not page Y when the formatting is the same?

**Comment by tfuxu** [comment]
I agree, I've updated title and issue description to focus only on disabling zoom in custom pages.
